index.js

```diff
@@ -4,7 +4,41 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+    let command = process.argv[2]
+    const numbers = []
+    let total = 0
+    let count = 0
 
+    // Edge cases that return errors when an invalid command is given
+    if (command !== 'plus' && command !== 'minus') {
+        if (command === undefined) {
+            return 'No operation provided...'
```

Comment:
If i run `node run.js 1 2 3`, I get `Invalid operation: 1`. How can you update this conditional to make sure you get the correct error message?